docs/user/using-an-OIDC-provider.md

```diff
@@ -359,3 +361,16 @@ Another common point of confusion is the `--oidc-username-prefix` option specifi
 ### Checking the logs of your Kubernetes API server
 
 Finally, if none of the above are relevant to your issue, you can check the logs of the Kubernetes API server deployment for OIDC-related lines at the time of your login attempt. These may show a configuration issue with the API server itself.
+
+### User automatically logged out from Kubeapps Console
+
+When using the default auth proxy, some users may experience the behavior where they are automatically logged out from the console.
+The default auth proxy (prior to version 7.0.5) is not configured to refresh the access/openid token and the console will logout once the token expires. In the case of Keycloak for example, this can happen quickly as the default access token expiration is 5mn.
```

Comment:
```suggestion
Prior to Kubeapps 7.0.5, the auth proxy configuration did not include a default `--cookie-refresh` value to refresh the access/openid token and so the console will logout once the token expires. In the case of Keycloak for example, this can happen quickly as the default access token expiration is 5m.
```